### ISSUE TITLE
Add commit message linter

### ISSUE THREAD

**Issue by moorereason** [bytesparadise/libasciidoc#700]
Enforce Angular Commit Message Guidelines within GitHub.

**Comment by xcoulon** [comment]
I need to see if such a linter already exist (as a GitHub Action would be ideal)

**Comment by moorereason** [comment]
The most popular GH Action appears to be https://github.com/wagoid/commitlint-github-action.

There's a simple GH App here: https://github.com/zeke/semantic-pull-requests.

**Comment by xcoulon** [comment]
cool, thanks for the links @moorereason! I'll give it a try soon

**Comment by gdamore** [comment]
Is there some meaningful tooling that consumes these commit messages?  TBH I really find them to be kind of a pain -- I've not seen them used anywhere else before.

**Comment by xcoulon** [comment]
yes, I generate the changelog that appears on https://github.com/bytesparadise/libasciidoc/releases using https://github.com/conventional-changelog/conventional-changelog and it parses the commit message to determine the type commit it was, and the scope.
